addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -234,7 +234,177 @@ class GVAR(actions) {
         allowSelfTreatment = 0;
         callbackSuccess = QFUNC(checkResponse);
     };
-
+    
+    // - Airways --------------------------------------------------------------
+    class ChestSeal: BasicBandage {
+        displayName = CSTRING(Actions_ChestSeal);
+        displayNameProgress = CSTRING(Actions_ChestSealProgress);
+        icon = "";
+        category = "airway";
+        treatmentLocations = TREATMENT_LOCATIONS_ALL;
+        allowedSelections[] = {"Body"};
+        allowSelfTreatment = 1;
+        medicRequired = 0;
+        treatmentTime = "5";
+        condition = QFUNC(canChestSeal);
+        callbackSuccess = QFUNC(chestSeal);
+        consumeItem = 1;
+        litter[] = {};
+        items[] = {"ACE_chestSeal"};
+    };
+    
+    class HeadTilt: BasicBandage {
+        displayName = CSTRING(Actions_HeadTilt);
+        displayNameProgress = CSTRING(Actions_HeadTiltProgress);
+        icon = "";
+        category = "airway";
+        treatmentLocations = TREATMENT_LOCATIONS_ALL;
+        allowedSelections[] = {"Head"};
+        allowSelfTreatment = 1;
```

Comment:
I shouldn't be able to tilt my on head (I think this doesn't make any difference)